Create 08-03-23_writeup.md (#203)
very-good-science/data-ethics-club: site/write_ups/2023/08-03-23_writeup.md

How does open source requirements for transparency clash with product requirements for success?
If something is purely open source, transparency is the only context with which to build trust. If you have a different framework then that can sometimes build trust inherently. The open source licencing model might help ensure transparency and code reuse.

In corporate and community projects, the corporate projects tend to be more task orientated and have clearer deliverables. Trade secrets would be more of an issue in industry and profits rely upon this. This might not be as clean of a product based on external requirements versus labor of love. We probably also need to tie some of deepfake materials to laws that protect individuals such as defamation laws.

> People will always find new ways to abuse technology as new technologies come about, but consequences of abuse has to tie back to laws that protect people and organisations. GDPR and Data Protection Act 2018 can help with protecting individuals privacy as a start but probably not enough to protect against deepfakes. By its nature, enforcement is already after the fact and quite clunky and hard to make use of, particuarly in such rapidly developing fields. 

Reviewer: ```suggestion
People will always find new ways to abuse technology as new technologies come about, but the consequences of abuse have to tie back to laws that protect people and organisations. The GDPR and Data Protection Act 2018 can help with protecting individuals' privacy as a start but probably not enough to protect against deepfakes. By its nature, enforcement is already after the fact and quite clunky and hard to make use of, particuarly in such rapidly developing fields. 
```

If Google released an open source deepfake tool, our sense was that there would be public outcry.
[Behavioural use licensing](https://en.wikipedia.org/wiki/Self-licensing) means in practise that as groups become more confident they tend to think less about the consequences of subsequent immoral behaivour. People contravene/break licenses all the time. Licenese and contracts might stop some bad use and that can still be meaningful but they are not a catch all solution. Companies still do a lot of unethical things despite licenses dictating what is/is not allowed.

> In the case of deep fakes, it tends to be a male dominated field. This is in stark contrast to the fact that the majority of people affected by the misuse of this are women. If it's open source, you have to rely on personal stakes for emotional accountability whereas with corperate structure you have more rigidity and guidelines. We probably also need to tie some of deepfake materials to laws that protect individuals such as defamantion laws for example.

Reviewer: ```suggestion
In the case of deep fakes, it tends to be a male dominated field. This is in stark contrast to the fact that the majority of people affected by the misuse of this are women. If it's open source, you have to rely on personal stakes for emotional accountability whereas with corperate structure you have more rigidity and guidelines.
```
Repetition :)

## Why do we think conversations about Ethics in AI often focus on design principles, such as Fairness, Accountability, or Transparency, instead of what systems are used for? 

There is a potential to lose context around analysis and a reason to hold back on sharing the data. This highlights different risks and governance. In contrast, something created by a community can have various purposes, corporate would have own purposes in mind. Would corporations go for the big public sector contracts if they were told they had to share their code?

How do we judge this? Case by case probably, as it's very difficult to guess context. 

A measure could be how easy is it to abuse? Without the code being shared does it slow you down in creating something that could do harm? "Ethical" varies from person to person and is changable. People in the interviews from the paper clashed.

To what extent are we putting a monetary value on harm by making it something that's legally enforceable? Is paying the fine going to be worth it for doing harm? Is that just paying the fee to do something unethical?

> Ethics seems to be more focussed on novelty. AI is new and exciting but we have so many things still to consider in data science. When we are at the beginning of building the tool which is a good time to think about this. But tools might not have a fixed, determined purpose so it's a constant guessing game of what tech can be used for, as well as a catchup game as new developments appear.

Reviewer: ```suggestion
Ethics seems to be more focussed on novelty. AI is new and exciting but we have so many things still to consider in data science. When we are at the beginning of building the tool is a good time to think about this. But tools might not have a fixed, determined purpose so it's a constant guessing game of what tech can be used for, as well as a catchup game as new developments appear.
```
